node/actors/network/src/testonly.rs

```diff
@@ -195,25 +177,42 @@ impl InstanceRunner {
     }
 }
 
+/// InstanceConfig
+pub struct InstanceConfig {
+    /// cfg
+    pub cfg: Config,
+    /// block_store
+    pub block_store: Arc<BlockStore>,
+    /// batch_store
+    pub batch_store: Arc<BatchStore>,
+    /// attestation_state
+    pub attestation_state: Arc<attestation::StateWatch>,
```

Comment:
As far as I see with the removal of the loop in run, nothing will keep this up-to-date, right? It would be helpful to leave a comment of what to expect, as previously that loop simulated having a client to pull data, in this case from the local batch store, so as long as the stores agreed there was a chance that they would all be voting on the same thing, but now I'm not sure what's the deal with this.

Reply:
I've added expected usage to the documentation of controller.

Reply:
I've updated the example with more details. Added second loop.